src/components/TodoComponents/TodoList.js

```diff
@@ -1,2 +1,23 @@
-// your components will all go in this `component` directory.
-// feel free to change this component.js into TodoList.js
+import React, { Component } from 'react'
+
+export class TodoList extends Component {
+    getStyle = () => {
+        return {
+            textDecoration: this.props.todo.completed ? 'line-through' : 'none'
+            } 
+        }
+    
+
+    
+    render() {
+        const { id,title } = this.props.todo;
+    return (
+    <div style={this.getStyle()} onClick={this.props.markComplete.bind(this, id)}>
```

Comment:
Same as your code, but this is a more common pattern:
```suggestion
    <div style={this.getStyle()} onClick={() => this.props.markComplete.(id)}>
```